data/html5video-rules.json

```diff
@@ -0,0 +1,32 @@
+{
+    "rules": [
```

Comment:
Is it possible to have single rule for all video formats?
As I can see, this change is about single concern - represent direct video url as a video tag.